### ISSUE TITLE
Are there generation options?

### ISSUE THREAD

**Issue by vortextemporum** [lucidrains/stylegan2-pytorch#213]
I am on half way training my transparent image dataset with this repo (first of all, thank you for this cool library).

However, I can't see options to generate images with seeds (or interpolations) like I can do in StyleGan2 tensorflow library. What are my options to generate thousands of images?

Thank you

**Comment by aertist** [comment]
since it generates a .pkl file, you can train the model from this then do all sorts of things with that .pkl file in the tensorflow version

**Comment by lucidrains** [comment]
@vortextemporum there's a section dedicated to that! https://github.com/lucidrains/stylegan2-pytorch#advanced-use it's as easy as one command for each of what you want to do

**Comment by vortextemporum** [comment]
Thank you both for your responds!

> since it generates a .pkl file, you can train the model from this then do all sorts of things with that .pkl file in the tensorflow version

My trained model files are .pt files. So does it mean that I should convert them back .pkl? Also, original Stylegan2 didn't allow for transparent images. If I just convert the model directly to pkl would it mess up?

> @vortextemporum there's a section dedicated to that! https://github.com/lucidrains/stylegan2-pytorch#advanced-use it's as easy as one command for each of what you want to do

I checked this section and using the commands in both here and lightweight-gan, but I don't see an option to set a seed range (like 10-8000) on generation, nor for interpolation.

**Comment by aertist** [comment]
> Thank you both for your responds!
> 
> > since it generates a .pkl file, you can train the model from this then do all sorts of things with that .pkl file in the tensorflow version
> 
> My trained model files are .pt files. So does it mean that I should convert them back .pkl? Also, original Stylegan2 didn't allow for transparent images. If I just convert the model directly to pkl would it mess up?
> 
> > @vortextemporum there's a section dedicated to that! https://github.com/lucidrains/stylegan2-pytorch#advanced-use it's as easy as one command for each of what you want to do
> 
> I checked this section and using the commands in both here and lightweight-gan, but I don't see an option to set a seed range (like 10-8000) on generation, nor for interpolation.

Check out this [repo](https://github.com/dvschultz/stylegan2-training) and his stiff on YT. It's really helpful in the context of this problem
